tico/experimental/quantization/passes/insert_quantize_on_dtype_mismatch.py

```diff
@@ -119,8 +119,7 @@ def _insert_quantize_op_before(node, inp):
             assert qparam.zero_point is not None
             scale = qparam.scale[0]
             zerop = qparam.zero_point[0]
-            min_ = qparam.min[0] if qparam.min is not None else 0
-            max_ = qparam.max[0] if qparam.max is not None else 0
+            min_, max_ = quant_min_max(qparam.dtype)
```

Comment:
Note that min/max of `torch.ops.quantized_decomposed.quantize_per_tensor.default` is quantized dtype's min/max, not the activation's min/max.

Reply:
Isn't it natural to set min/max of `QuantParam` when it is created instead? And, these codes here wouldn't have to be changed.

Reply:
`min_` and `max_` are used for `torch.ops.quantized_decomposed.quantize_per_tensor.default`, not QuantParam.

Reply:
QuantParam's min/max is activation min/max, which is different from quant_min and quant_max.

Reply:
Ah, I just thought about introducing `quant_min` and `quant_max`, which seems more flexible to me. Of course we've got no case where qmin/qmax and dtype are aligned though. 

Current codes looks good to me as well.

Reply:
That could be another design choice :) Maybe we can save `quant_min` and `quant_max` instead of `dtype` in QuantParam. I just used `dtype` because `quant_min` and `quant_max` are not used in circle.